lib/loadClients.js

```diff
@@ -22,7 +22,10 @@ function applyToClient (client, method) {
 var TREASURE_KEYS = [
   'init',
   'set',
+  'get',
```

Comment:
These API are left out intentionally because they a synchronous APIs. When SDK users call these APIs, they expect results immediately, which is not the cases for these loader methods, which in turn can mislead SDK users